src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -158,9 +283,44 @@ const GroupCheckbox = styled(Box)`
   }
 `;
 
-const StyledCount = styled(Count)`
+const StyledPrimaryCount = styled(Count)`
```

Comment:
nit: I would just call this `PrimaryCount` - we use `Styled-` as a prefix generally when we don't have a better/more specific name.

Reply:
will do